jscomp/main/melc_cli.ml

```diff
@@ -398,6 +402,27 @@ module Internal = struct
     let doc = "*internal* <file> Suffix for interface files (default: .mli)" in
     Arg.(value & opt (some string) None & info [ "intf-suffix" ] ~doc)
 
+  let g =
+    let doc = "*internal* Save debugging information" in
+    Arg.(value & flag & info [ "g" ] ~doc)
+
+  let opaque =
+    let doc =
+      "*internal* <file> Does not generate cross-module optimization \
+       information (reduces necessary recompilation on module change)"
+    in
+    Arg.(value & flag & info [ "opaque" ] ~doc)
+
+  let strict_sequence =
```

Comment:
This one and `strict_formats` below should default to `true`, like they do here: https://github.com/melange-re/melange/blob/d2112f44cc8e630f64d0b6138edd553de280adc3/jscomp/core/bs_conditional_initial.ml#L57

Reply:
I am not sure I understand. The logic in the PR is setting them to true if the flag is passed, but it honors whatever defaults are set "upstream" (either in melange-compiler-libs first, or the overrides that happen in `setup_env` in melange afterwards).

Ftr, in OCaml trunk, one is `true` and the other `false`, not sure if it'd be better to move towards that direction: https://github.com/ocaml/ocaml/blob/9a267738ca7311c3da39c491f71b4047ea884620/utils/clflags.ml#L90-L91.

Reply:
ah you're handling it elsewhere, I missed that before!

Reply:
Some history:

- https://forum.rescript-lang.org/t/more-strict-checks-over-toplevel-expressions/581
- https://github.com/rescript-lang/rescript-compiler/pull/4385